IntelSiliconPkg/Feature/SmmAccess/Library/PeiSmmAccessLib/PeiSmmAccessLib.c

```diff
@@ -108,7 +108,7 @@ Close (
 {
   SMM_ACCESS_PRIVATE_DATA  *SmmAccess;
   BOOLEAN                  OpenState;
-  UINT8                    Index;
+  UINT8                    Index;   // MU_CHANGE - Use consistent width in loop comparison

```

Comment:
Is this necessary?

Reply:
That was a mistake. I messed up changing the type when adding the comment. I'll fix it, thanks.